velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -505,6 +505,10 @@ class PlanBuilder {
       const std::string& idName = "unique",
       const int32_t taskUniqueId = 1);
 
+  PlanBuilder& markDistinct(
+      const std::vector<std::shared_ptr<const core::FieldAccessTypedExpr>>&
```

Comment:
Please, document this method and change input to std::vector<std::string> to allow for passing column names like in other methods.